runtime/jit_vm/cthelpers.cpp

```diff
@@ -29,6 +29,10 @@
 
 extern "C" {
 
+#if JAVA_SPEC_VERSION >= 20
```

Comment:
Minor nit, maybe a question for @TobiAjila. 

Should this be `>= 21` since JDK 20 is no longer available and we no longer build it?  Or do we specify it this way because that's when the annotation was first introduced and its good for bookkeeping purposes?  I'm fine either way, just curious.

Reply:
Actually, the annotation appears in JDK 19 as well it seems.  So if we're back-leveling the JDKs for bookkeeping purposes then it should be at least JDK 19.

Reply:
For new code, we typically don't account for out-of-support releases. So we can ignore jdk19 and jdk20.

Reply:
I changed the JAVA_SPEC_VERSION check to 21. Once the tests you launched passes, I can squash and rebase.